crates/sui-framework/packages/deepbook/sources/clob.move

```diff
@@ -829,7 +834,7 @@ module deepbook::clob {
         clock: &Clock,
         account_cap: &AccountCap,
         ctx: &mut TxContext
-    ): (u64, u64, bool, u64) {
+    ): (u64, u64, bool, Option<u64>) {
```

Comment:
This approach is unnecessary and may increase the cost of using the code it calls.

Reply:
To maintain backward compatible, we have revert back now.